Add removeReviewedInSprint command

bparees opened the pull request:
Won't compile until @eparis lands his other changes to add the new flag constants.

bparees committed:
Add removeReviewedInSprint command

bparees commented on cmd/bug-automation/generate/main.go:
```diff
@@ -258,6 +280,13 @@ func doGenerate() error {
+			Query:       bugsWithReviewedInSprintFlagQuery(),
+			Update:      bugsWithReviewedInSprintFlagUpdate(),
+			Default:     false,
+		},
```
i made this a new command rather than adding logic to the existing command because it was easier given the structure of things.  I figure for whatever duration we have overlap, we can just run both remove commands, unless you think that's too expensive.

eparis replied:
Can you update the Makefile to run both?

eparis replied:
oh, you need me to compile, harumph.

eparis replied:
I created https://github.com/openshift/bugzilla-tools/pull/33   after we merge that we need to run the generator, update the Makefile to run both and merge this.

bparees replied:
i rebased on your PR, updated the makefile, and ran the generator and added the new yaml file.

eparis committed:
Change UpcomingSprint to use ReviewedInSprint Flag

This will look for either UpcomingSprint keyword or the ReviewedInSprint Flag.
Eventually we can drop the keyword portion.
